packages/wdio-browserstack-service/tests/launcher.test.ts

```diff
@@ -9,6 +9,8 @@ const expect = global.expect as unknown as jest.Expect
 const log = logger('test')
 const error = new Error('I\'m an error!')
 const sleep = (ms: number = 100) => new Promise((resolve) => setTimeout(resolve, ms))
+// @ts-ignore
```

Comment:
This should not be needed.
```suggestion
```

Reply:
Had added this as importing from `package.json` throws the `'rootDir' is expected to contain all source files.` error.